README.md

```diff
@@ -1,98 +1,169 @@
+$ pip install git+https://github.com/cfpb/flask-eventics
+```
 
-**Example**
+Once installed, the `flask-eventics` blueprint simply needs to be
+registered with your flask application:
 
-If you have questions, concerns, bug reports, etc, please file an issue in this repository's Issue Tracker.
+```python
+from flask import Flask
+from flask_eventics import eventics
 
-## Getting involved
+app = Flask(__name__)
+app.register_blueprint(eventics)
+```
 
-This section should detail why people should get involved and describe key areas you are
-currently focusing on; e.g., trying to get feedback on features, fixing certain bugs, building
-important pieces, etc.
+If using [Sheer](https://github.com/cfpb/sheer), `flask-eventics` can be
+added to your site's `_settings/blueprints.json` file.
 
-General instructions on _how_ to contribute should be stated with a link to [CONTRIBUTING](CONTRIBUTING.md).
+```json
+{
+  "flask_eventics": {
+    "package": "flask_eventics",
+    "module":  "eventics"
+  }
+}
+```
 
+## Configuration
 
-----
-
-## Open source licensing info
-1. [TERMS](TERMS.md)
-2. [LICENSE](LICENSE)
+`flask-eventics` can optionally use the Flask app configuration or
+environment variables for configuration. 
+
+#### `EVENT_ICS_URL` 
+The URL at which ICS files will be available. It should include a named 
+argument, `event_slug`, which will corropsond to an event available 
+from a source url.
+
+Example: `/events/<event_slug>/ics`
+
+#### `EVENT_SOURCE` 
+The REST URL at which JSON that describes a particular event is available. 
+This was designed to work with Elasticsearch, but it should work with any 
+REST API that provides all the necessary properties within a single 
+JSON object.
+
+Example: `http://localhost:9200/content/event/<event_slug>/_source`
+
+#### `EVENT_CALENDAR_PRODID` 
+The iCalendar product identifier that will be returned for generated 
+calendars.
+
+#### JSON-iCalendar Field Mapping
+
+There is a series of configuration options that map iCalendar fields to
+properties on the JSON object returned by the event source. 
+
+* `EVENT_FIELD_SUMMARY`: `summary`
+* `EVENT_FIELD_DTSTART`: `dtstart`
+* `EVENT_FIELD_DTEND`: `dtend`
+* `EVENT_FIELD_DTSTAMP`: `dtstamp`
+* `EVENT_FIELD_UID`: `uid`
+* `EVENT_FIELD_PRIORITY`: `priority`
+* `EVENT_FIELD_ORGANIZER`: `organizer`
+* `EVENT_FIELD_ORGANIZER_ADDR`: `organizer_email`
+* `EVENT_FIELD_LOCATION`: `location`
+* `EVENT_FIELD_STATUS`: `status`
+
+#### Field Defaults
+
+The following configuration options define defaults that will be used 
+if the above fields are not available in the JSON object returned by 
+the event source, 
+
+* `EVENT_DEFAULT_SUMMARY`: ``,
+* `EVENT_DEFAULT_DTSTART`: `2015-01-11T10:30:00Z`,
+* `EVENT_DEFAULT_DTEND`: `2015-01-11T10:30:00Z`,
+* `EVENT_DEFAULT_DTSTAMP`: `2015-01-11T10:30:00Z`,
+* `EVENT_DEFAULT_UID`: ``,
+* `EVENT_DEFAULT_PRIORITY`: 1,
+* `EVENT_DEFAULT_ORGANIZER`: ``,
+* `EVENT_DEFAULT_ORGANIZER_ADDR`: ``,
+* ` EVENT_DEFAULT_LOCATION`: ``,
```

Comment:
Extra space here. Also, empty backticks render out as visible backticks. Is that okay with you?